shell.core/src/main/java/org/metaborg/spoofax/shell/client/Repl.java

```diff
@@ -47,6 +37,7 @@
         this.display = display;
         this.invoker = invoker;
 
+        // TODO: this does not really belong here.
```

Comment:
> yea I agree. In fact we are still not sure whether IEditor should have a setPrompt and setContinuationPrompt method.

> That being said, I also think there's nothing wrong with exposing hooks to shell.console / shell.eclipse that allow the user to change the look of the prompt :p

To elaborate, this is the reason I moved these methods to the interface in the first place: setting a default in Repl to get a uniform look, allowing users to recognize Spoofax Shell by its prompt, and then exposing a function to change these defaults.